src/actions/user.actions.js

```diff
@@ -9,21 +9,17 @@ export const userActions = {
 
 // Actions creator, the alerts will be removed
 function login(email, password) {
-    return dispatch => {
+    return async dispatch => {
         dispatch(request({ email }));
-
-        userService.login(email, password)
-            .then(
-                user => { 
-                    dispatch(success(user));
-                    history.push("/home");
-                    dispatch(alertActions.success("Login successful")); //This will be removed
-                },
-                error => {
-                    dispatch(failure(error.toString()));
-                    dispatch(alertActions.error(error.toString())); //This will be removed
-                }
-            );
+        try {
+            const user = await userService.login(email, password)
+            dispatch(success(user));
+            history.push("/home");
```

Comment:
In general we do it inside the React login component, the success action should set a flag somewhere(usually in redux but I noticed you aren't use it), so then you consume it in your Login component and check it out, if it's true you redirect the user to the homepage. Suppose user was in the homepage and they go to the Login page, they must be redirected to the homepage again because they are already logged in.

Reply:
Great! I'll do it in the next PR